addons/website_sale/controllers/main.py

```diff
@@ -777,6 +777,8 @@ def cart(self, access_token=None, revive='', **post):
             'date': fields.Date.today(),
             'suggested_products': [],
         })
+        additional_values = request.env['sale.order'].get_website_sale_extra_values(order)
+        values.update(additional_values)
```

Comment:
```suggestion
        values.update(order._get_cart_rendering_values))
```
Why an `api.model` method if you always have an order, you can give it directly as `self` ;)
Also, keep methods private unless you really need to make them public.